builtin/docker/pull/kaniko.go

```diff
@@ -79,16 +80,23 @@ func (b *Builder) pullWithKaniko(
 		// is "index.docker.io".
 		host = "index.docker.io"
 	}
-	log.Trace("auth host", "host", host)
 
+	var insecure bool
```

Comment:
Instead of the bool here for using it later, could we rely on `ai.Insecure` again?